marklogic-data-hub-central/ui/src/config/guided-tour-steps.js

```diff
@@ -21,27 +21,27 @@ const browseTableViewSteps = [
   },
   {
     selector: '[id*="sideBarContainer"]',
-    content: 'Here are the available faceted search options. Entity Properties are per selected entity. Hub Properties are per document.'
+    content: 'Here are the available faceted search options. Entity Properties are per selected entity. Hub Properties are per record.'
   },
   {
     selector: '[id*="snippetView"]',
     content: 'You are currently in table view. Click on this button to switch to snippet view'
   },
   {
     selector: '.ant-table-row',
-    content: 'Document results include entity name, primary key (or document URI), table information, and metadata'
+    content: 'Record results include entity name, primary key (or record URI), table information, and metadata'
   },
   {
     selector: '.ant-table-row-expand-icon-cell',
-    content: 'Expansion button will show all the properties of each document'
+    content: 'Expansion button will show all the properties of each record'
```

Comment:
I think we can say "entity" here, but "record" is still safe, as "record" is always a safe generalization.